src/main/kotlin/eu/kevin/api/models/payment/initiatePayment/request/InitiatePaymentRequest.kt

```diff
@@ -0,0 +1,24 @@
+@file:UseSerializers(BigDecimalSerializer::class)
+
+package eu.kevin.api.models.payment.initiatePayment.request
+
+import eu.kevin.api.serializers.BigDecimalSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.UseSerializers
+import java.math.BigDecimal
+
+@Serializable
+data class InitiatePaymentRequest @JvmOverloads constructor(
+    val redirectUrl: String,
+    val amount: BigDecimal,
+    val currencyCode: String,
+    val description: String,
+    var bankPaymentMethod: BankPaymentMethod,
+    var cardPaymentMethod: CardPaymentMethod? = null,
```

Comment:
what if I want only to use **CardPaymentMethod**? I will still need to supply **BankPaymentMethod**?